_positions/TAM-cloud.gov.md

```diff
@@ -0,0 +1,43 @@
+
+---
+title: Technical Account Manager — Cloud.gov 
+permalink:
+team: Infrastructure
+active: false
+---
+
+###Role summary
+
+As a technical account manager for the Cloud.gov, you’ll act as a product ambassador to our partners, representing products to to them and ensuring they can easily use them. Additionally, you’ll ensure that our partners maintain a strong relationship with the 18F team. The 18F team and its partners will rely on you to assist with product understanding, documentation, and increasing business. 
+
+This position is actually a hybrid between a technical account manager and a business account manager. We’re looking for someone who can talk tech and business with clients, the 18F team, and stakeholders — someone who has experience in government or highly regulated industries (such as financial markets, health, and so on). This position is best suited to someone with considerable experience in documentation, sales, and product management.
+
+
+###Key objectives
+
+**Objective \#1: Be a strong ambassador of the product.**
+- Have a full understanding of the product; deftly answer inquiries about it and describe how the product will exceed the partner’s  needs.
+- Determine whether a given product is right for the partner; help them understand why there is (or isn’t) a fit. 
+- Always provide a clear value proposition explanation to make a sale. 
+- Demonstrate technical insight: Understand the technical capabilities of the product to be able to answer questions on what the product can and cannot do. 
+
+
+
+**Objective \#2: Practice an exceptional level of customer service with all partners, providing a unique, tailored experience.**
+- Explain the product to people who have varying levels of technical knowledge — always meet the client where they’re at. 
+- Express empathy through the long compliance and security processes.  
+- Skillfully map specific inquiries to product capabilities, identifying the product that best meets the partner’s needs.
+- Finalize partnerships with other government agencies
+- Serve as a liaison between the clients and the team: Provide client feedback to the team, enabling the team to make necessary changes to product performance or presentation.
+
+
+
+**Objective \#3: Translate difficult technical terms and concepts so that they’re accessible to all audiences.**
+- Present the technical details of a product in such a manner that clients can purchase the product and start using it without assistance. 
+- Communicate clearly, calmly, and concisely in all situations. 
+- Create compelling case studies on how users are benefiting from cloud.gov, providing a special focus on lessons learned. 
+- Leverage inquires about the product to create useful resources for the public, including blog posts, FAQs, one sheets, contributions to community forums, technical processes, and more.
+
+
+
+##Apply to the
```

Comment:
Flagging that this will need an application link before merging. The `Styling with Markdown is supported` link has examples of how to include a link.